frontend/src/features/landing_page/api/getImages.ts

```diff
@@ -0,0 +1,34 @@
+import { createNextcloudAPI } from '@/config/axios';
+
+export const getPhotos = async (): Promise<Array<string>> => {
```

Comment:
Great to see that but it would probably be better to have this in a different PR. It definitely needs some documentation what it is for.

Reply:
Yes, as mentioned in some other PR. The purpose was for testing requests with credentials against the NextcloudAPI.
I can add a comment mentioning this, so we can improve on it in a dedicated issue.

Reply:
Yes, write that at the top of the file. Then everything is clear.

Reply:
There is probably a better location for it though.

Reply:
I created a separate issue and branch for this: #364 
and moved the functions there